modules/ROOT/pages/addition/docker/container.adoc

```diff
@@ -1,21 +1,55 @@
-= Monitor instance in standalone Docker container   
+= Monitor a Neo4j instance in standalone Docker container
 
-Running additional processes that are not tied to the lifecycle of the container process is not a best practice for containerized applications.
-Nevertheless, it is possible to transparently monitor a containerized Neo4j instance in following ways:
+A Neo4j instance running in a Docker container can be monitored by a NOM agent in two ways:
+ 
+- NOM agent can be run as an additional (daemon) process in the Neo4j container
+- NOM agent can be run on the Docker host to externally monitor the Neo4j container
 
 [NOTE]
 ====
-Both methods below use a bundled NOM agent inside the Neo4j image instead of downloaded agent packages.
-Replacing `products/neo4j-ops-manager-agent-*-linux-amd64.tar.gz` with path to NOM agent package downloaded should give the same results.
+Running additional processes that are not tied to the lifecycle of the container process is not a best practice for containerized applications.
 ====
 
-. Monitoring a Neo4j instance Running inside a Docker container:
-+
+== In-container Monitoring
+Following steps describe running a NOM agent in Neo4j container as an additional (daemon) process to monitor the instance:
+
+* Run the Neo4j docker container-
+    ** This https://neo4j.com/docs/operations-manual/current/docker/[operations] document describe various ways to run a Neo4j instance docker container
+    ** A custom entrypoint script can be mounted to Neo4j container to run the NOM agent at DBMS startup which is described <<entrypoint, below>>
+    
+* Install NOM agent - to make `agent` command available to run
+    ** Bundled
+    NOM agent binaries are bundled in the Neo4j package and reside in the `products` folder of the Neo4j installation directory. It can be
+    extracted from the package and installed into a system binaries directory for access to `agent` command.
```

Comment:
```suggestion
    NOM agent binaries are bundled in the Neo4j package and reside in the `products` folder of the Neo4j installation directory. 
    It can be extracted from the package and installed into a system binaries directory for access to `agent` command.
```